index.html

```diff
@@ -13,7 +28,18 @@
     Name: <input id="name" type="text" name="fullname"><br>
     <input id="submitButton" type="submit" value="Submit">
 
-
+<div class="radio">
+    <label><input id="nfl" type="radio" name="optradio" unchecked>NFL</label>
+</div>
+<div class="radio">
+    <label><input id="nba" type="radio" name="optradio" uncheked>NBA</label>
+</div>
+<div class="radio disabled">
+    <label><input id="nhl" type="radio" name="optradio" unchecked>MLB</label>
+</div>
+<div class="radio">
+    <label><input id="mlb" type="radio" name="optradio" unchecked>NHL</label>
+</div>
```

Comment:
Not sure if we want to have these in alphabetical order or not.   Also, do we want to have these appear "in-line" so that it doesn't take up too much vertical space on the page?

Reply:
I was under the impression that I was just getting this in to help keep the ball rolling with regard to your and Nancy's work. It doesn't matter to me how we order them and I will of course be formatting them in a way that is smarter than the way it presents now. Just haven't gotten there yet.

Reply:
Correct.  I am simply making note of it here as a comment so we don't forget to possibly alphabetize it later.  Could you add a value attribute equal to the league name?